lib/Service/PageService.php

```diff
@@ -41,7 +41,8 @@ public function __construct(private IAppManager $appManager,
 		private UserFolderHelper $userFolderHelper,
 		private IUserManager $userManager,
 		private IConfig $config,
-		ContainerInterface $container) {
+		ContainerInterface $container,
+		private SessionService $sessionService) {
```

Comment:
Should we inject the Iqueue here too

Reply:
IQueue might be unavailable if the notify_push app is not installed. That's why it's taken from the container with exception handling.

Reply:
work for >28 https://github.com/nextcloud/server/pull/41265